clients/web/src/stylesheets/body/adminConsole.styl

```diff
@@ -2,7 +2,7 @@ ul.g-admin-options
   list-style-type none
   font-size 18px
   padding-left 0
-  maring 0
```

Comment:
What, haven't you guys heard of the CSS4 `maring` property? It's like margin, only a lot prettier.